src/Factory/RefundPaymentFactory.php

```diff
@@ -15,30 +15,25 @@
 
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Model\PaymentMethodInterface;
-use Sylius\Component\Core\Repository\PaymentMethodRepositoryInterface;
-use Sylius\RefundPlugin\Entity\RefundPayment;
 use Sylius\RefundPlugin\Entity\RefundPaymentInterface;
 
 final class RefundPaymentFactory implements RefundPaymentFactoryInterface
 {
-    /** @var PaymentMethodRepositoryInterface */
-    private $paymentMethodRepository;
+    /** @var string */
+    private $className;
 
-    public function __construct(PaymentMethodRepositoryInterface $paymentMethodRepository)
+    public function __construct(string $className)
     {
-        $this->paymentMethodRepository = $paymentMethodRepository;
+        $this->className = $className;
     }
```

Comment:
This change is not announced in changelog

Reply:
Added 👌🏻